content/applications/inventory_and_mrp/inventory/routes/strategies/removal.rst

```diff
@@ -1,222 +1,352 @@
-with the products fitting your settings the best.
+Next, several orders for the same product are made, but in this example, the goods weren't received
+on the same day and they don't have the same expiration date. In that situation, logically, sending
+those with the closest expiration date is preferred. Depending on the chosen removal strategy, Odoo
+generates a transfer with the products that fit the settings best.
 
 .. image:: removal/packing-products.png
    :align: center
-   :alt: products being packed at packing area for delivery, taking expiration dates into account.
+   :alt: Products being packed at the packing area for delivery, taking expiration dates into
+         account.
 
 .. note::
-   On the transfer form, you can find the product’s lot/serial number to pick for delivery.
+   To pick for delivery, the product's lot/serial number can be found on the transfer form.
+
+How each removal strategy works
+===============================
 
-How does it work?
-=================
+Removal strategies determine which products are taken from the warehouse when orders are confirmed.
+This is why selecting the correct removal strategies to best suit the needs of the warehouse is
+important.
 
 First In, First Out (FIFO)
 --------------------------
 
-When using a **First In, First Out** strategy, a demand for some products triggers a removal rule
-which requests a transfer for the lot/serial number that has entered your stock the first.
+When using a :guilabel:`First In, First Out (FIFO)` strategy, a demand for a product triggers a
+removal rule, which requests a transfer for the lot/serial number that entered the stock first.
+For example, imagine there are three lots of nails in the warehouse. Those three have the following
+lot numbers: 00001, 00002, 00003, each with five boxes of nails in it.
 
-To be clearer, let’s imagine that you have three lots of nails in your warehouse. Those three have
-the following lot numbers: 00001, 00002, 00003, each with 5 nails boxes in it. 00001 entered the
-stock on the 23rd of May, 00002 on the 25th of May, and 00003 on the 1st of June. A customer orders
-you 6 boxes on the 11th of June. With the *FIFO* strategy selected, a transfer is requested for the
-five boxes of 00001 and one of the boxes in 00002 because 00001 has entered your stock before the
-others. The box from 00002 is taken because it has the oldest enter date after 00001.
+Lot 00001 entered the stock on May 23, lot 00002 on May 25, and lot 00003 on June 1. A customer
+orders six boxes on June 11.
+
+Using the :abbr:`FIFO (First In, First Out)` removal strategy, a transfer is requested for the five
+boxes from lot 00001 and one of the boxes in lot 00002, since lot 00001 entered the stock first. The
+box from lot 00002 is taken next because it has the oldest receipt date after lot 00001.
+
+So, for every order of a product with the FIFO strategy selected, Odoo requests a transfer for the
```

Comment:
```suggestion
So, for every order of a product with the :abbr:`FIFO (First In, First Out)` strategy selected, Odoo requests a transfer for the
```
I'd keep the abbreviation tags consistent when referencing the removal strategies